src/DripsHub.sol

```diff
@@ -339,11 +356,11 @@ abstract contract DripsHub {
                 split += splitsAmt;
                 address splitsReceiver = currReceivers[i].receiver;
                 splitsStates[calcUserId(splitsReceiver)].collectables[DUMMY_ASSET] += splitsAmt;
-                emit Split(user, splitsReceiver, splitsAmt);
+                emit Split(user, splitsReceiver, DUMMY_ASSET, splitsAmt);
```

Comment:
We also need to move `splitting part` to an own function and call it here. 

I guess this will happen in the upcoming PRs.

Reply:
All in its own time, but you're right, we'll do that too :+1: